src/kafka/Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM confluentinc/cp-kafka:latest-ubi8
 
+USER root
```

Comment:
Should we allow using the `root` user? I remember that we modify all service user to non-root user in previous pr.

@open-telemetry/demo-approvers @open-telemetry/demo-maintainers

Reply:
The root permissions are needed for the chmod of the agent. They are dropped again later down in the Dockerfile.